docs/add-ins/addin-tutorial.md

```diff
@@ -501,15 +596,22 @@ function buildFileList(files) {
 }
 ```
 
-That fully implements the settings dialog. Now the question is how do we invoke it? You may have noticed that we did not add a button to the ribbon for settings. Instead, the add-in will check that it has been configured. If it hasn't, then it will prompt the user when they invoke the add-in to configure before proceeding. Since the user could click either button first, we'll do this check in both cases.
+> [!NOTE]
+> You may have noticed that there's no button to invoke the settings dialog. Instead, the add-in will check whether it has been configured when the user chooses either the **Insert default gist** button or the **Insert gist** button. If the add-in has not yet been configured, the settings dialog will prompt the user to configure before proceeding. 
+
+## Implement a UI-less button
+
+This add-in's **Insert default gist** button is a UI-less button that will invoke a JavaScript function, rather than open a task pane like many add-in buttons do. When the user chooses the **Insert default gist** button, the corresponding JavaScript function will check whether the add-in has been configured. 
 
-### Implementing a UI-less button
+- If the add-in has already been configured, the function will load the content of the gist that the user has selected as default and insert it into the body of the message. 
```

Comment:
Include "the" before "default"
Suggest: ...user has selected as the default

Reply:
Good suggestion; will change.